CHANGELOG.md

```diff
@@ -84,4 +84,8 @@ new TextStyleBuilder()
 - New/Break : #383 Get a callback when the image source is touched `onTouchSourceImage(MotionEvent event);`
 
 ### 2.0.0
-- New : Migrated the app and library to Kotlin
\ No newline at end of file
+- New : Migrated the app and library to Kotlin
+
+### 2.1.0-alpha01
```

Comment:
We don't have an alpha stage, so we can directly bump it to `2.1.0.`. This applies to all the files where we have used alpha versioning.

Reply:
Fixed